content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,77 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [-t <name_or_path>]
+                      [--targets [<path> [<path> ...]]] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>]
+                      [--x-label <text>] [--y-label <text>]
+                      [revisions [revisions ...]]
 
 positional arguments:
-  revisions             Git commits to plot from
+  revisions             Git commits to plot from/to
 ```
 
 ## Description
 
-This command visualize difference between metrics among experiments in the
-repository history. Requires that Git is being used to version the metrics
-files.
+This command is a way to visualize the difference between metrics among
+experiments in the <abbr>repository</abbr> history. The target metric files
+(required) should specified with the `--targets` (`-t`) option.
```

Comment:
`be specified`

Reply:
should -> must

Reply:
No, actually targets are no longer required. Updated in a6132335.

I thought I had updated this before...